challenge.rb

```diff
@@ -21,8 +26,13 @@ def is_of_age? age, minimum_age
 # This question will be graded independently of question one.
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
+
 def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+  if is_of_age?(age, drinking_age) == true
```

Comment:
The idiom `x == true` is redundant; if the expression evaluates as true, it must be because `x` is true (and vice versa). In other words, it's the same as just writing `x` - or, in this case, 
`if is_of_age?(age, drinking_age)`